cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -208,6 +212,23 @@ func textFormatPrintMetric(obj *api.Metric) {
 	fmt.Printf("%s: %s | Expire: %s\n", peer.IDB58Encode(obj.Peer), obj.Value, date)
 }
 
+func textFormatPrintRepoGC(obj *api.RepoGC) {
+	if obj.Error != "" {
+		fmt.Printf("  > Error: %s\n", obj.Error)
```

Comment:
should return?

Reply:
No, it should just print errors saved in api.RepoGC to the terminal